File: leetcode/fizzbuzz.py
from typing import List

class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        l  = []
        j = n
        for i in range(j):
            n = i + 1
            if n % 15 == 0:
                l.append("FizzBuzz")
            elif n % 5 == 0:
                l.append("Buzz")
            elif n % 3 == 0:
                l.append("Fizz")
            else:
                l.append(str(n))
        return l

File: leetcode/fizz-buzz.cpp
#include <vector>
#include <string>

class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> a;
        for(int i = 1; i < n+1; ++i) {
            if (i % 15 == 0) {
                a.push_back("FizzBuzz");
            } else if (i % 5 == 0) {
                a.push_back("Buzz");
            } else if (i % 3 == 0) {
                a.push_back("Fizz");
            } else {
                a.push_back(std::to_string(i));
            }
        }
        return a;
    }
};
